web/crux-ui/src/server/kratos.ts

```diff
@@ -8,25 +8,40 @@ import { missingParameter } from './error-middleware'
 
 const config = new Configuration({ basePath: process.env.KRATOS_URL })
 const kratos = new V0alpha2Api(config)
+const meta = new MetadataApi({
+  basePath: 'http://172.17.0.1:9434',
```

Comment:
Wouldn't be better to use an env variable?

Reply:
uh nice catch ty. totally forgot that one